README.md

```diff
@@ -373,16 +385,21 @@ Otherwise, they will be resent at the next interval.
     >>
     >> [
     >>   {
-    >>     topic: <name>,
-    >>     type:  <type>,
-    >>     url:   <url>,
-    >>     t:     <t>
+    >>     topic: <string>,
+    >>     type:  <string>,
+    >>     url:   <string>,
+    >>     t:     <integer>,
+    >>     data:  <anything>
```

Comment:
Hmmm. Indeed. And interestingly, the spec that intends to test for too-large blobs actually tests that passing a string fails (irrespective of the size). Needs fixing.